src/syscall_sdl.c

```diff
@@ -276,3 +337,458 @@ void syscall_submit_queue(riscv_t *rv)
         }
     }
 }
+
+/* Portions Copyright (C) 2021-2022 Steve Clark
+ *
+ * This software is provided 'as-is', without any express or implied
+ * warranty.  In no event will the authors be held liable for any damages
+ * arising from the use of this software.
+ *
+ * Permission is granted to anyone to use this software for any purpose,
+ * including commercial applications, and to alter it and redistribute it
+ * freely, subject to the following restrictions:
+ *
+ * 1. The origin of this software must not be misrepresented; you must not
+ *    claim that you wrote the original software. If you use this software
+ *    in a product, an acknowledgment in the product documentation would
+ *    be appreciated but is not required.
+ * 2. Altered source versions must be plainly marked as such, and must not be
+ *    misrepresented as being the original software.
+ * 3. This notice may not be removed or altered from any source distribution.
+ */
+
+/*
+ * This is a straightforward MUS to MIDI converter made for programs
+ * like DOOM that use MIDI to store sound.
+ *
+ * Sfx_handler can handle Quake's sound effects because they are all in WAV
+ * format.
+ */
+
+typedef struct {
+    char id[4];
+    uint16_t score_len;
+    uint16_t score_start;
+} __attribute__((packed)) mus_header_t;
```

Comment:
Wrap `__attribute__((packed))` via the macro defined in `src/common.h. Consider Win32/MSVC compatibility.